config/schemas/mod.json

```
@@ -78,6 +112,26 @@
 			"type":"boolean",
 			"description": "If set to true, mod will not be enabled automatically on install"
 		},
+		
+		"english" : {
+			"$ref" : "#/definitions/localizeable"
```

in other places as well
```suggestion
			"$ref" : "#/definitions/localizable"
```